src/plugins/drawing-tool/objects/drawing-object.ts

```diff
@@ -41,6 +42,15 @@ export const DrawingObject = types.model("DrawingObject", {
 })
 .views(self => ({
   get boundingBox(): BoundingBox {
+    // SC: I tried an approach of tracking the SVG elements that were rendered,
+    // and using a generic SVG getBBox() here, but it had performance issues.
```

Comment:
fwiw, getBBox used to force a reflow of the page so on a complicated page, if it happened with regularity (e.g. while the user was dragging something) it would slow things down quite a bit. I'm not sure when these variable chips get resized... but that could have been why?

Reply:
Thanks! I didn't know that. Yes it was being called as the user dragged things around. The size wasn't really changing, but the x y position was. According to this: https://gist.github.com/paulirish/5d52fb081b3570c81e3a getBBox still causes a reflow. So that was probably it.